lib/Client.php

```diff
@@ -155,21 +157,48 @@ public function isFeatureEnabled(
      * @param string $distinctId
      * @param mixed $defaultValue
      * @param array $groups
+     * @param array $personProperties
+     * @param array $groupProperties
      * @return bool | string
      * @throws Exception
      */
     public function getFeatureFlag(
         string $key,
         string $distinctId,
         bool $defaultValue = false,
-        array $groups = array()
+        array $groups = array(),
+        array $personProperties = array(),
+        array $groupProperties = array()
```

Comment:
needs two more params: onlyEvaluateLocally (default false) , and sendFeatureFlagEvents (default true)